docs/user-guide/ecosystem.md

```diff
@@ -6,68 +6,83 @@ On this page you can find a non-exhaustive list of libraries and tools that supp
 
 ### Table of contents:
 
-- [Apache Arrow](#apache-arrow)
-- [Data visualisation](#data-visualisation)
-- [IO](#io)
+- [Database integration](#database-integration)
+- [Data pipelines](#data-pipelines)
+- [Interchange and open storage formats](#interchange-and-storage-formats)
 - [Machine learning](#machine-learning)
-- [Other](#other)
+- [Validation and quality](#validation-and-quality)
+- [Visualization and presentation](#visualization-and-presentation)
 
 ---
 
-### Apache Arrow
+### Database integration
 
-[Apache Arrow](https://arrow.apache.org/) enables zero-copy reads of data within the same process, meaning that data can be directly accessed in its in-memory format without the need for copying or serialisation. This enhances performance when integrating with different tools using Apache Arrow. Polars is compatible with a wide range of libraries that also make use of Apache Arrow, like Pandas and DuckDB.
-
-### Data visualisation
+#### DuckDB
 
-#### hvPlot
+[DuckDB](https://duckdb.org) is a high-performance analytical database system. It is designed to be fast, reliable, portable, and easy to use. DuckDB provides a rich SQL dialect, with support far beyond basic SQL. DuckDB supports arbitrary and nested correlated subqueries, window functions, collations, complex types (arrays, structs), and more. Read about integration with Polars [on the DuckDB website](https://duckdb.org/docs/guides/python/polars).
 
-[hvPlot](https://hvplot.holoviz.org/) is available as the default plotting backend for Polars making it simple to create interactive and static visualisations. You can use hvPlot by using the feature flag `plot` during installing.
+#### LanceDB
 
-```python
-pip install 'polars[plot]'
-```
+[LanceDB](https://lancedb.com/) is a developer-friendly, serverless vector database for AI applications. They have added a direct integration with Polars. LanceDB can ingest Polars dataframes, return results as polars dataframes, and export the entire table as a polars lazyframe. You can find a quick tutorial in their blog [LanceDB + Polars](https://blog.lancedb.com/lancedb-polars-2d5eb32a8aa3)
```

Comment:
Nitpick but should it be "Polars" or "polars" (or it doesn't matter)? I'm always switching between the two in my docs but some consistency would be good :)

Reply:
Good find. It should be Polars. I'll fix it when I am back at my computer.